shoes-core/lib/shoes/color/dsl_helpers.rb

```diff
@@ -81,7 +83,11 @@ def image_file?(arg)
       end
 
       def image_pattern(path)
+        path = absolute_file_path(path)
         Shoes::ImagePattern.new path if File.exist?(path)
+      rescue FileNotFoundError => e
+        Shoes.logger.error(e.message)
+        return nil
```

Comment:
To be honest, this was just matching the existing spec'd behavior. It feels in line with Shoes' historical defaults that it just omits the background if it fails to find the file. Will ponder and dig a bit further.